### DISCUSSION TITLE
multihop Mesh networking

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [yggdrasil-network/yggdrasil-go#940] [Q&A] [answered]
How to enable [multi-hop routing](https://en.wikipedia.org/wiki/Multi-hop_routing) mesh networks in Yggdrasil
![image](https://user-images.githubusercontent.com/42504385/188256233-4064e312-18a0-40b4-a9dc-13dda22d90d3.png)

**Comment by neilalexander** [comment, marked as answer]
Yggdrasil is multi-hop by default. You don’t need to enable it specifically.
